inception/inception-curation/src/main/java/de/tudarmstadt/ukp/inception/curation/settings/CurationProjectSettingsMenuItem.java

```diff
@@ -50,7 +53,13 @@ public IconType getIcon()
     @Override
     public String getLabel()
     {
-        return "Curation";
+        IRequestablePage currentPage = PageRequestHandlerTracker.getLastHandler(RequestCycle.get())
```

Comment:
We have a new static method `de.tudarmstadt.ukp.clarin.webanno.support.wicket.resource.Strings.getString(String)` that should be used in backend code like this. However, it requires that the string to be obtained is contained in a `wicket-package.properties` file in the package or parent-package of the code that calls the method.

I am using this new method in very few places at the moment, e.g. in `de.tudarmstadt.ukp.inception.annotatorjs.AnnotatorJsHtmlAnnotationEditorFactory.getDisplayName()`.